src/test/java/utilities/Util.java

```diff
@@ -1,7 +1,22 @@
 package utilities;
 
-public class Util {
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
 
+public class Util {
+	
+    /*
+     * @Author: surayya Abraham
+     * 
+     */
+	public static void dragAndDropAcions(WebDriver driver, String sourceXpath , String targetXpath) {
+		Actions action = new Actions(driver);
+		WebElement drage = driver.findElement(By.xpath("sourceXpath"));
+		WebElement drop = driver.findElement(By.xpath("targetXpath"));
+		 action.dragAndDrop(drage, drop).perform();
```

Comment:
1, It would be a better idea to use By object instead of xpath, would be more reusable
2, Your parameters (sourceXpath, targetXpath) are not actually used 
3, Fix your formatting